woocommerce/woocommerce-android
# Edit variations: Inventory

This PR adds the inventory editing functionality to Variation details. The current Inventory screen functionality has been extracted from `ProductDetailViewModel` into a separate one so that it could be reused for variations.

![image](https://user-images.githubusercontent.com/1522856/90663198-a495eb80-e249-11ea-8d79-43235bbb2626.png)

**To test:**
1. Go to Products -> Tap on a product
2. Tap on the Inventory property
3. Verify the inventory data is shown as expected
4. Change some value
5. Notice the Done button appears
6. Tap on the back button
7. Notice that a discard dialog is shown
8. Tap Cancel
9. Tap on the Done button
10. Verify that the inventory information was updated in product details
11. Tap on Update
12. Verify that the product data was updated
13. Go back to Products -> Tap on a variable product -> Tap on Variations -> Tap on a variation item
14. Tap on the Inventory property
15. Notice that the "Limit one per order" switch is _not_ shown
16. Repeat steps 3-12 for the variation
